data/default.json

```diff
@@ -60,6 +62,2854 @@
+      "userType": "light",
+      "geolocation": {
+        "range": [2186870784, 2186936319],
+        "country": "NL",
+        "region": "OV",
+        "eu": "1",
+        "timezone": "Europe/Amsterdam",
+        "city": "Enschede",
+        "ll": [52.22, 6.9009],
+        "metro": 0,
+        "area": 100
+      },
+      "createdAt": "2019-05-21T08:38:18.479Z",
+      "updatedAt": "2019-05-21T08:38:18.479Z",
+      "__v": 0
+    },
+    {
+      "_id": "5ce3f1de9a0ed300329a8d0b",
+      "user": "5c055c1083e11e0211c24ad4",
+      "userId": "5c055c1083e11e0211c24ad4",
+      "sessionId": null,
+      "visibility": "private",
+      "space": "5b56e70ab253099933364412",
+      "spaceId": "5b56e70ab253099933364412",
+      "verb": "navigate",
+      "postedAt": "2019-05-21T12:41:02.385Z",
+      "sessionSequenceNumber": 4,
+      "userType": "light",
+      "geolocation": {
+        "range": [2186870784, 2186936319],
+        "country": "NL",
+        "region": "OV",
+        "eu": "1",
+        "timezone": "Europe/Amsterdam",
+        "city": "Enschede",
+        "ll": [52.22, 6.9009],
+        "metro": 0,
+        "area": 100
+      },
+      "createdAt": "2019-05-21T12:41:02.526Z",
+      "updatedAt": "2019-05-21T12:41:02.526Z",
+      "__v": 0
+    },
+    {
+      "_id": "5d29fea1570dbc001a68f3c6",
+      "user": "5c055c1083e11e0211c24ad4",
+      "userId": "5c055c1083e11e0211c24ad4",
+      "sessionId": null,
+      "visibility": "public",
+      "space": "5b56e70ab253099933364412",
+      "spaceId": "5b56e70ab253099933364412",
+      "verb": "navigate",
+      "postedAt": "2019-07-13T15:54:08.501Z",
+      "sessionSequenceNumber": 3,
+      "userType": "light",
+      "geolocation": {
+        "range": [3577206784, 3577208831],
+        "country": "CH",
+        "region": "",
+        "eu": "0",
+        "timezone": "Europe/Zurich",
+        "city": "",
+        "ll": [47.1449, 8.1551],
+        "metro": 0,
+        "area": 100
+      },
+      "createdAt": "2019-07-13T15:54:09.796Z",
+      "updatedAt": "2019-07-13T15:54:09.796Z",
+      "__v": 0
+    },
+    {
+      "_id": "5d2db1b8590563001a54e547",
+      "user": "5c055c1083e11e0211c24ad4",
+      "userId": "5c055c1083e11e0211c24ad4",
+      "sessionId": null,
+      "visibility": "public",
+      "space": "5b56e70ab253099933364412",
+      "spaceId": "5b56e70ab253099933364412",
+      "verb": "navigate",
+      "postedAt": "2019-07-16T11:15:04.314Z",
+      "sessionSequenceNumber": 3,
+      "userType": "graasp",
+      "geolocation": {
+        "range": [2159255552, 2159256575],
+        "country": "CH",
+        "region": "VD",
+        "eu": "0",
+        "timezone": "Europe/Zurich",
+        "city": "Lausanne",
+        "ll": [46.516, 6.6328],
+        "metro": 0,
+        "area": 50
+      },
+      "createdAt": "2019-07-16T11:15:04.220Z",
+      "updatedAt": "2019-07-16T11:15:04.220Z",
+      "__v": 0
+    },
+    {
+      "_id": "5ca467b2bf8a9500322dc14e",
```

Comment:
@juancarlosfarah What is the benefit of an action without a verb?

Reply:
Actions don't require a `verb`. Often they have one, but if they don't, we will still store them.